src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -1113,67 +1101,50 @@ public static function updateCategory(array $item, array $meta = null): int
         return $updated;
     }
 
-    /**
-     * Update an existing comment
-     *
-     * @param array $item The new data.
-     *
-     * @return int
-     */
-    public static function updateComment(array $item): int
+    public static function updateComment(array $item)
     {
-        // update category
-        return BackendModel::getContainer()->get('database')->update(
-            'blog_comments',
-            $item,
-            'id = ?',
-            [(int) $item['id']]
+        $entityManager = BackendModel::get('doctrine.orm.default_entity_manager');
```

Comment:
idem

Reply:
fixed in cd9cef81ef540cf410bc31750966a00591aa06ae